group_vars/matrix_servers

```diff
@@ -4307,6 +4309,7 @@ matrix_synapse_container_labels_public_client_root_redirection_enabled: "{{ matr
 matrix_synapse_container_labels_public_client_root_redirection_url: "{{ (('https://' if matrix_playbook_ssl_enabled else 'http://') + matrix_server_fqn_element) if matrix_client_element_enabled else '' }}"
 
 matrix_synapse_container_labels_public_client_synapse_admin_api_enabled: "{{ matrix_synapse_admin_enabled }}"
+matrix_synapse_container_labels_internal_client_synapse_admin_api_enabled: "{{ matrix_bot_draupnir_admin_api_enabled }}"
```

Comment:
The `matrix_synapse_container_labels_internal_client_synapse_admin_api_enabled` variable is written to here, but it's not defined in `defaults/main.yml` in the `matrix-synapse` role.

There should be a definition with some default value (`false`) there.